packages/obonode/obojobo-chunks-list/viewer-component.scss

```diff
@@ -17,3 +17,15 @@
 		}
 	}
 }
+
+.li-spacing-moderate {
+	li {
+		margin: 0.5em 0;
+	}
+}
+
+.li-spacing-generous {
+	li {
+		margin: 1em 0;
+	}
+}
```

Comment:
Same with the editor-component.scss, these should be more specific - they need to be inside the root .obojobo-draft--chunks--list style as well. When all of the CSS gets compiled together by webpack the result is that all of the individual scss files are smashed together into one big file. Since these are by themselves not defined in any rule it means they'll impact anything on the page with the class of "li-spacing-generous". Unlikely to happen since it's a unique name, but want to avoid any chance of styles doing unexpected things.

Reply:
Additionally, reviewing the display of each of the list options in the viewer, they all look good except the moderate and generous options push the surrounding content around the list farther away. Adding some first-child and last-child rules to kill the top and bottom margins should fix it.

Compact:

![image](https://user-images.githubusercontent.com/73479/117361823-83813380-ae88-11eb-9129-6153f4433c0e.png)

Generous, note the two text lines above and below the list are father away.

![image](https://user-images.githubusercontent.com/73479/117361870-909e2280-ae88-11eb-92bf-f4645dce7091.png)